### PR TITLE
name change et al

### PR DESCRIPTION
**Pull request by doktordirk**
…thentication' and references to 'aurelia-api' without 'spoonx/' prefix

### CODE REVIEW

**Comment by RWOverdijk** [comment]
@doktordirk A breaking change usually has a body (in stead of being a one-liner).

so something like this: `git commit -m` (no body)

Then:

```
feat(project): Rename project to remove spoonx prefix

BREAKING CHANGE: `spoonx/` prefix dropped from install name, for authentication and api. Update `package.json` and `config.js` accordingly.
```

**Comment by doktordirk** [comment]
ah. i tried (something) like that, but didn't get into the changelog. must have had the wrong keywords or something

**Comment by RWOverdijk** [comment]
@doktordirk Maybe we should do a "ready for review" comment. Or set a "ready for review" label on the issue. I'm not sure when it is ready for review now :)

**Comment by doktordirk** [comment]
yup. though i'd prefer two categories 'please comment' (ignore formating, not intended to be merged yet) and 'please review' (aka supposably  mergable).

**Comment by RWOverdijk** [comment]
@doktordirk Created the labels. Have fun with them :D

**Comment by doktordirk** [comment]
@RWOverdijk and this

File: README.md [hunk 1]
```diff
@@ -5,7 +5,12 @@
 
 > Makes setting up authentication for your Aurelia app simple.
 
+## Important note
+
+This plugin is now registered with npm and jspm as `aurelia-authentication`. This simplifies the installation. With this change `aurelia-authentication` will reference to [aurelia-api](https://github.com/spoonx/aurelia-api) as `aurelia-api` only (loosing the `spoonx/..` prefix). You will need to update your project's package.json accordingly to prevent forks.
```
Comment (RWOverdijk) [inline README.md]: Same simplification here (as the other PR)

**Comment by RWOverdijk** [comment]
These doc changes should be moved, too. They're not in the docs yet so for now the changes can stay in this PR.

TL;DR; :+1:

**Pull request merged**